### PR TITLE
2-Regression/1-Tools/translations/README.es.md

### PR DESCRIPTION
**Pull request by davidit33**

### CODE REVIEW

**Review by alfredodeza — CHANGES_REQUESTED** [review]

**Review by davidit33 — COMMENTED** [review]
It's Ok now?

File: 2-Regression/1-Tools/translations/README.es.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+
+   > Siéntase cómodo con Python trabajando con esta colección de [módulos de aprendizaje](https://docs.microsoft.com/users/jenlooper-2911/collections/mp1pagggd5qrq7?WT.mc_id=academic-15963-cxa)
+
+3. **Instale Scikit-learn**, siguiendo [estas instrucciones](https://scikit-learn.org/stable/install.html). Dado que debe asegurarse de usar Python3, se recomienda que use un entorno virtual. Tenga en cuenta que si está instalando esta biblioteca en una Mac M1, hay instrucciones especiales en la página vinculada arriba.
+
+1. **Instale Jupyter Notebook**. Deberá [instalar el paquete de Jupyter](https://pypi.org/project/jupyter/). 
+
+## El entorno de creación de ML
+
+Va a utilizar **cuadernos** para desarrollar tu código Python y crear modelos de machine learning. Este tipo de archivos es una herramienta común para científicos de datos, y se pueden identificar por su sufijo o extensión `.ipynb`.
```
Comment (alfredodeza) [inline 2-Regression/1-Tools/translations/README.es.md:39]: Not trying to nitpick, but this is still going from formal (_Va a utilizar_) to informal (_desarrollar tu código_).

Comment (davidit33) [reply 2-Regression/1-Tools/translations/README.es.md]: To have the opportunity to learn from people like you, it's a blessing, please be as demanding as possible with my work, I want to learn and be successful. Thanks!

Comment (davidit33) [reply 2-Regression/1-Tools/translations/README.es.md]: You can contact me and share any information that helps me do a better job.

File: 2-Regression/1-Tools/translations/README.es.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+En esta lección, aprenderá a:
+
+- Configurar su computadora para tares locales de machine learning.
+- Trabajar con cuadernos Jupyter.
+- Usar Scikit-learn, incluida la instalación.
+- Explorar la regressión lineal con un ejercicio práctico.
+
+## Instalaciones y configuraciones.
+
+[![Uso de Python con Visual Studio Code](https://img.youtube.com/vi/7EXd4_ttIuw/0.jpg)](https://youtu.be/7EXd4_ttIuw "Uso de Python con Visual Studio Code")
```
Comment (alfredodeza) [inline 2-Regression/1-Tools/translations/README.es.md:21]: We recently updated this to use a video specific to this lesson (link is updated in the English version). Could you be so kind and update the link? I think the captions/text can stay the same.

```suggestion
[![Uso de Python con Visual Studio Code](https://img.youtube.com/vi/yyQM70vi7V8/0.jpg)](https://youtu.be/yyQM70vi7V8 "Uso de Python con Visual Studio Code")
```

**Comment by alfredodeza** [comment]
This looks good to go. One tiny comment about the VSCode video and I think we can merge this. Thanks so much!

**Pull request merged**